src/utils/systemIntake.ts

```diff
@@ -39,3 +40,18 @@ export const translateRequestType = (requestType: RequestType) => {
       return '';
   }
 };
+
+/**
+ * Get the acronym representation of the component name
+ * @param componentToTranslate - component name string that needs to be converted to an acronym
+ */
+export const getAcronymForComponent = (componentToTranslate: string) => {
+  const component = cmsDivisionsAndOffices.find(
+    c => c.name === componentToTranslate
+  );
+  if (component) {
```

Comment:
I think we can just have the rest be

```suggestion
  return component ? component.acronym : 'Other'
```